ArduPlane/mode_loiter.cpp

```diff
@@ -25,3 +27,56 @@ void ModeLoiter::update()
     plane.calc_throttle();
 }
 
+bool ModeLoiter::isHeadingLinedUp(const Location loiterCenterLoc, const Location targetLoc)
+{
+    const uint16_t loiterRadius = abs(plane.aparm.loiter_radius);
+    if (loiterCenterLoc.get_distance(targetLoc) < loiterRadius + loiterRadius*0.05) {
+        /* Whenever next waypoint is within the loiter radius plus 5%,
+           maintaining loiter would prevent us from ever pointing toward the next waypoint.
+           Hence break out of loiter immediately
+         */
+        return true;
+    }
+
+    // Bearing in centi-degrees
+    const int32_t bearing_cd = plane.current_loc.get_bearing_to(targetLoc);
+    return isHeadingLinedUp(bearing_cd);
+}
+
+bool ModeLoiter::isHeadingLinedUp(const float bearing)
```

Comment:
the float/int overload here is going to be much to tight, and cause people to get this wrong. The float one should just be removed, and the other one should be renamed possibly as far as `isHeadingLinedUp_cd` or something, to make it clearer.

As an aside I can't find anything that calls this version, can we just remove it?

Reply:
Removed this version and renamed the other function as you suggested.